README.md

```diff
@@ -43,9 +43,9 @@ database: metastore
 username: quickwit
 assword: ""
 ```
-Configure `QW_METASTORE_URI` in `extraEnvFrom` instead (see
+You can use `config.metastore_uri` or `QW_METASTORE_URI` in `extraEnvFrom` instead (see
 [documentation](https://quickwit.io/docs/configuration/metastore-config) for
-more details).
+more details). `metastore_uri` can be built with the old variables like this: `postgres://$(POSTGRES_USERNAME):$(POSTGRES_PASSWORD)@$(POSTGRES_HOST):$(POSTGRES_PORT)/$(POSTGRES_DATABASE)`.
```

Comment:
Why refer to this as "old variables" ? Because they seems aren't mentioned somewhere anymore.

I'd replace "old" with "already defined environment variables on the pod" which can take other names btw s: `postgres://$(PG_USER):$(PG_PASSWORD)@$(PG_HOST):$(PG_PORT)/$(PG_DATABASE)`.

It'll work regardless of the name if those environment variables are defined on the pod with `extraEnv`.